pkgs/test/test/utils.dart

```diff
@@ -5,22 +5,32 @@
 import 'dart:async';
 import 'dart:collection';
 
+import 'package:boolean_selector/boolean_selector.dart';
+import 'package:glob/glob.dart';
+import 'package:test_api/scaffolding.dart' // ignore: deprecated_member_use
```

Comment:
We have an import to `package:test/test.dart` which should expose `Timeout`. Why do we need this import?